logging/unit_tests/test_client.py

```diff
@@ -92,11 +96,10 @@ def test_no_gax_ctor(self):
 
     def test_sinks_api_wo_gax(self):
         from google.cloud.logging._http import _SinksAPI
-        from google.cloud.logging import client as MUT
-        from google.cloud._testing import _Monkey
 
-        with _Monkey(MUT, _USE_GAX=False):
-            client = self._make_one(self.PROJECT, credentials=_Credentials())
+        client = self._make_one(
+            self.PROJECT, credentials=_Credentials(),
+            use_gax=False)
 
```

Comment:
ISTM that this should really be testing the environment sniff, rather than the explicit constructor parameter.